### ISSUE TITLE
Question about following process

### ISSUE THREAD

**Issue by kimwongyuda** [lanyunshi/ConversationalKBQA#3]
![image](https://user-images.githubusercontent.com/35097915/146138427-a2d38576-0957-4d7d-a2ac-f34c1287b744.png)

when computing focal scores, do parameters w(t) and b(t) compute unnormalized focal score for each entity and softmax computes normalized focal scores with inputs of concatenation of unnormalized focal scores?

**Comment by lanyunshi** [comment]
Yes, you are right. Softmax here is to get the distribution over all the entities based on their scores.
